pages/index.js

```diff
@@ -11,118 +11,185 @@ export default function Home() {
     >
       <div className="container">
         <main>
-          <h2> Connecting Furloughed Healthcare Workers with Healthcare Facilities in Need </h2>
-          <p>
-            <strong> Louisiana Health Work Connect</strong> is an initiative organized by
-            the State of Louisiana Department of Health in response to COVID-19 to help facilities
-            with acute healthcare staffing shortages fill those vacancies with qualified candidates
-            who are otherwise out of work.
-          </p>
+          <h3>
+            <strong> Louisiana Health Work Connect</strong> is an initiative
+            organized by the State of Louisiana Department of Health in response
+            to COVID-19 to help facilities with acute healthcare staffing
+            shortages fill those vacancies with qualified candidates who are
+            otherwise out of work.
+          </h3>
+
+          <div style={{ width: "100%", marginTop: 18 }}>
+            <h4>How does it work? </h4>
+            <ol>
+              <li>
+                Interested health care workers submit their information and
+                qualifications.{" "}
+              </li>
+              <li>
+                Participating healthcare facilities regularly update the
+                Louisiana Department of Health (LDH) on their staffing needs.{" "}
+              </li>
+              <li>
+                LDH provides a list of candidates to facilities tailored to
+                their staffing needs.{" "}
+              </li>
+              <li>
+                A participating facility then contacts a health care worker to
+                begin the hiring process.{" "}
+              </li>
+            </ol>
+          </div>
+
           <div className="grid">
             <Link href="/candidate-intake?source=Landing%20Page">
-              <a className="card">
```

Comment:
Changing this a back to a div means the main CTA is no longer in the tab order -> not keyboard accessible